src/components/dialogs/load-creation-dialog.js

```diff
@@ -0,0 +1,483 @@
+ * @param {String} title Title of the dialog
+ */
+const LoadCreationDialog = ({ open, onClose, network }) => {
+    const classes = useStyles();
+
+    const studyUuid = decodeURIComponent(useParams().studyUuid);
+
+    const intl = useIntl();
+    const intlRef = useIntlRef();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const [busOrBusbarSectionOptions, setBusOrBusbarSectionOptions] = useState(
+        []
+    );
+
+    const [loadId, setLoadId] = useState('');
+
+    const [loadName, setLoadName] = useState('');
+
+    const [loadType, setLoadType] = useState('');
+
+    const [activePower, setActivePower] = useState('');
+
+    const [reactivePower, setReactivePower] = useState('');
+
+    const [voltageLevel, setVoltageLevel] = useState('');
+
+    const [busOrBusbarSection, setBusOrBusbarSection] = useState('');
+
+    const [errors, setErrors] = useState(new Map());
+
+    const handleChangeLoadId = (event) => {
+        setLoadId(event.target.value);
+    };
+
+    const handleChangeLoadName = (event) => {
+        setLoadName(event.target.value);
+    };
+
+    const handleChangeLoadType = (event) => {
+        setLoadType(event.target.value);
+    };
+
+    const handleChangeActivePower = (event) => {
+        // Replace ',' by '.' to ensure double values can be parsed correctly
+        setActivePower(event.target.value?.replace(',', '.'));
+    };
+
+    const handleChangeReactivePower = (event) => {
+        // Replace ',' by '.' to ensure double values can be parsed correctly
+        setReactivePower(event.target.value?.replace(',', '.'));
+    };
+
+    const handleChangeVoltageLevel = (event, value, reason) => {
+        if (reason === 'select-option') {
+            setVoltageLevel(value);
+            setBusOrBusbarSection('');
+            if (value?.topologyKind === 'NODE_BREAKER') {
+                // TODO specify the correct network variant num
+                fetchBusbarSectionsForVoltageLevel(studyUuid, 0, value.id).then(
+                    (busbarSections) => {
+                        setBusOrBusbarSectionOptions(busbarSections);
+                    }
+                );
+            } else if (value?.topologyKind === 'BUS_BREAKER') {
+                // TODO specify the correct network variant num
+                fetchBusesForVoltageLevel(studyUuid, 0, value.id).then(
+                    (buses) => setBusOrBusbarSectionOptions(buses)
+                );
+            } else {
+                setBusOrBusbarSectionOptions([]);
+            }
+        } else if (reason === 'clear') {
+            setVoltageLevel(null);
+            setBusOrBusbarSection('');
+            setBusOrBusbarSectionOptions([]);
+        }
+    };
+
+    const handleChangeBus = (event, value, reason) => {
+        setBusOrBusbarSection(value);
+    };
+
+    const handleSave = () => {
+        let tmpErrors = new Map(errors);
+
+        tmpErrors.set(
+            'load-id',
+            validateField(loadId, {
```

Comment:
It's a pity that validation occurs only when handling save button click but it seems defined like that in the story so ok.